torch/csrc/cuda/Module.cpp

```diff
@@ -1085,8 +1085,11 @@ static void registerCudaDeviceProperties(PyObject* module) {
           &cudaDeviceProp::name
 #endif // USE_ROCM
           )
-#ifndef FBCODE_CAFFE2
+#if (defined(USE_ROCM) && ROCM_VERSION >= 60100) || !USE_ROCM
```

Comment:
not sure if `!USE_ROCM` is the expressed intent here or rather just `defined(CUDA_VERSION)`

Reply:
This code shall be already compiled under cuda device. I've duplicated the line to evaluate if a right version of ROCm is used. In the past this code was failing to compiled within Meta, now with similar condition it does.